cleanlab_studio/studio/studio.py

```diff
@@ -144,3 +144,60 @@ def apply_corrections(self, cleanset_id: str, dataset: Any) -> Any:
             corrected_ds[label_column] = joined_ds["__cleanlab_final_label"]
             corrected_ds = corrected_ds[joined_ds["action"] != "exclude"]
             return corrected_ds
+
+    def create_project(
+        self,
+        dataset_id: str,
+        project_name: str,
+        *,
+        tasktype: str = "multi-class",
+        modality: Optional[str] = None,
+        modeltype: str = "regular",
+        label_column: Optional[str] = None,
+        feature_columns: Optional[List[str]] = None,
+        text_column: Optional[str] = None,
```

Comment:
since this is a fairly complex interface (with some non-obvious arguments) and is user-facing, we definitely need to add a docstring here. docstring should focus on the high-level meaning of each argument, from a user perspective